transport/conn.go

```diff
@@ -454,6 +455,16 @@ func (c *Conn) sendRequest(req frame.Request, compress, tracing bool) (frame.Res
 	return resp.Response, resp.Err
 }
 
+func (c *Conn) useKeyspace(keyspace string) error {
```

Comment:
Needs to be Exported.

Reply:
I'd suggest moving it after Startup, it's often the 1st thing you want to do post Startup.